features/admin.authentication-flow-builder.v1/data/flow-sequences/passwordless/push-auth-sequence.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This will enable push auth login sequence in visual editor, but may not show it in the classic editor (which is disabled in IS, but still enabled in Asgardeo).

Therefore, we may need to updated the classic editor quickstart as well.

![Screenshot 2025-01-29 at 12 10 22](https://github.com/user-attachments/assets/48f8de7b-4004-4a0b-a93e-64aba0549d3f)